packages/babel-helper-hoist-variables/package.json

```diff
@@ -13,11 +13,13 @@
     "access": "public"
   },
   "main": "./lib/index.js",
-  "TODO": "The @babel/traverse dependency is only needed for the NodePath TS type. After converting @babel/core to TS we can import NodePath from there.",
   "dependencies": {
-    "@babel/traverse": "workspace:^7.13.15",
     "@babel/types": "workspace:^7.13.16"
   },
+  "TODO": "The @babel/traverse dependency is only needed for the NodePath TS type. We can consider exporting it from @babel/core.",
+  "devDependencies": {
+    "@babel/traverse": "workspace:*"
```

Comment:
Traverse is only used for a type import:

https://github.com/babel/babel/blob/d3f4c22c28e19569d2330b70c48550508da86246/packages/babel-helper-hoist-variables/src/index.ts#L1-L3